src/models/signStringModels.ts

```diff
@@ -0,0 +1,17 @@
+/** SignString options used when contructing a signString object */
+export type SignStringOptions = {
```

Comment:
Rename all to SignMessage...

Reply:
At the moment, I have renamed files, classes, and fields. If we determine that the term "string" is more abstract and suitable for this context, I am open to reverting the changes back.